### DISCUSSION TITLE
Integration with Apple Health / Google Fit

### DISCUSSION THREAD

**Discussion by astashov** [astashov/liftosaur#117] [Ideas]
We could log all the finished workouts to Apple Health / Google Fit - sets, reps and weights. Also can log the bodyweight, and body measurements (I assume). And import the bodyweight from there into Liftosaur.

I'm not super familiar with the featureset of Apple Health / Google Fit yet, but I think something along those lines.

**Comment by AdEmergency625** [comment]
Updated the correct YT link. 

As I posted at Reddit https://www.reddit.com/r/liftosaur/comments/1f06zep/comment/ljtzxhd/?utm_source=share&utm_medium=web3x&utm_name=web3xcss&utm_term=1&utm_content=share_button, I would appreciate Apple Shortcut integration too. It supercharges the mightiness of this app.  See this youtube video. https://youtu.be/uazJnUX5Yng?si=pVQ8wcm3bEdJ8s7R

**Reply by astashov** [reply]
@AdEmergency625 I think you posted wrong url for the youtube video, do you have a right one?

**Reply by AdEmergency625** [reply]
Updated the original post.

Btw, being a Sales/Marketing professional, I would say the people who play with Apple Shortcut are the perfect target audience for Liftosaur. They would love the programmability and the flexibility of your app.

**Reply by astashov** [reply]
Thanks! Yeah, this is fair :)

**Comment by bpwhelan** [comment]
Just something to think about... In terms of Google Fit Integration, it seems everybody is moving towards supporting https://developer.android.com/health-and-fitness/guides/health-connect

Which kind of acts as a proxy between many different sources/destinations

**Reply by astashov** [reply]
Yeah, apparently you cannot even sign up for using Google Fit anymore, Google officially (and pretty forcefully :)) deprecates it.
I'm adding Google Health Connect integration.

**Reply by bpwhelan** [reply]
[Classic Google](https://killedbygoogle.com/) :)

But at least in this case i think Google Health Connect makes a ton more sense.

**Comment by astashov** [comment]
This is done
